File: modeler/lib/mes/modeler.rb
require 'jquery/rails'
require 'jquery-ui-rails'
require 'js_cookie_rails'
require 'jquery-hoverIntent-rails'
require 'select2-rails'
require 'sprockets/rails'
require 'bootstrap-sass'
require 'mes-core'
require 'mes/modeler/engine'

module Mes::Modeler
  mattr_accessor :modeler_path
  @@modeler_path = '/'
end

File: modeler/app/assets/javascripts/mes/modeler.js
//= require jquery
//= require js.cookie
//= require jquery.hoverIntent
//= require select2
//= require jquery_ujs
//= require jquery-ui/sortable
//= require bootstrap-sprockets
//= require mes/layout/modeler
//= require mes/modeler/workflows

$(document).ready(function() {
  $("select.select2").select2({
    theme: "bootstrap",
    width: 'resolve'
  });
});
